seatunnel-api/src/main/java/org/apache/seatunnel/api/table/type/SeaTunnelRow.java

```diff
@@ -237,7 +237,9 @@ private int getBytesForValue(Object v) {
             case "String[]":
                 int s = 0;
                 for (String i : ((String[]) v)) {
-                    s += i.length();
+                    if (null != i) {
+                        s += i.length();
+                    }
                 }
                 return s;
             case "Boolean[]":
```

Comment:
Thanks @fcb-xiaobo ! I think we need to abstract a general method to filter out null values ​​in all types of arrays. Not only `String[]` but `Boolean[]`,`Long[]` etc.

Reply:
ok，i try abstract common methon check all values is null